### PR TITLE
Run goimports on the whole project

### PR DESCRIPTION
**Pull request by metalmatze**
When opening this repository in my editor, I run goimports automatically. This is what it found.

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Comment by strk** [comment]
Could you please rebase and fix conflicts ?

**Comment by DblK** [comment]
Since master has changed, @metalmatze can you update your PR?
Thanks

**Comment by tboerger** [comment]
Please rebase as @strk mentioned.

File: modules/markdown/markdown_test.go [hunk 1]
```diff
@@ -1,13 +1,13 @@
 package markdown_test
 
 import (
-	. "github.com/gogits/gogs/modules/markdown"
-	. "github.com/smartystreets/goconvey/convey"
+	"bytes"
 	"testing"
 
-	"bytes"
+	. "github.com/gogits/gogs/modules/markdown"
```
Comment (bkcsoft) [inline modules/markdown/markdown_test.go]: nope

File: modules/markdown/markdown_test.go [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 	"github.com/gogits/gogs/modules/setting"
```
Comment (bkcsoft) [inline modules/markdown/markdown_test.go]: nope

File: models/repo_test.go [hunk 3]
```diff
@@ -1,11 +1,11 @@
 package models_test
 
 import (
-	. "github.com/gogits/gogs/models"
-	. "github.com/smartystreets/goconvey/convey"
 	"testing"
 
+	. "github.com/gogits/gogs/models"
```
Comment (bkcsoft) [inline models/repo_test.go]: This needs to be changed

File: models/repo_test.go [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 	"github.com/gogits/gogs/modules/markdown"
```
Comment (bkcsoft) [inline models/repo_test.go]: and this

**Review by bkcsoft — CHANGES_REQUESTED** [review]

**Comment by metalmatze** [comment]
Recreated the commit with the current `master`.
Wrong import paths are gone too. Sorry @bkcsoft.

**Pull request merged**